app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
Yeah ok it got lost in 649f04b482f813804dad178f5800846d8b3d64c5. I think in terms of Nov Matake's license, I believe the entire `controllers/api/openid_connect`, `models/api/openid_connect/`, and `lib/api/openid_connect` should be somehow "covered".